src/utils/system-validation.ts

```diff
@@ -0,0 +1,14 @@
+import chalk from "chalk";
+import { execa } from "execa";
+
+export const validateFoundryUp = async () => {
+  try {
+    await execa("foundryup", ["-h"]);
+  } catch (error) {
+    const message = `${chalk.bold.yellow("NOTE: Foundryup is not installed in your system.")}
```

Comment:
Maybe use `Attention:` instead of `NOTE:`?

Reply:
Ohh yes! makes more sense, update at https://github.com/scaffold-eth/create-eth/pull/65/commits/826ad68d3e62c110cbe7026e4c124bdb54ce4bb5